shesha-core/src/Shesha.Application/Notifications/Dto/DirectNotificationJobArgs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Shesha.Domain;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shesha.Notifications.Dto
+{
+    public class DirectNotificationJobArgs
```

Comment:
Please rename properties which contain identifiers to xxxId, e.g. FromPerson -> FromPerson**Id**